presto-docs/src/main/sphinx/security/ldap.rst

```diff
@@ -0,0 +1,212 @@
+===================
+LDAP Authentication
+===================
+
+Presto can be configured to enable frontend LDAP authentication over
+HTTPS for clients, such as the :ref:`cli_ldap`, or the JDBC and ODBC
```

Comment:
Should we mention about the JDBC/ODBC drivers here in the docs here, given that the open source drives don't have LDAP support?